lib/src/components/NxTabs/NxTabs.scss

```diff
@@ -42,26 +42,58 @@
   margin: 0 var(--nx-spacing-4x) -1px 0;
   // subtract border thickness for consistent spacing
   padding: var(--nx-spacing-4x) var(--nx-spacing-6x) calc(var(--nx-spacing-4x) - 2px) var(--nx-spacing-6x);
+  position: relative;
   text-align: center;
   min-width: 80px;
   max-width: 240px;
+  background-color: transparent;
+  opacity: 99%;
 
   &.active {
     @include nx-text-helpers.semi-bold;
 
-    border-bottom-color: var(--nx-swatch-blue-40);
+    border-bottom: 2px solid var(--nx-swatch-blue-40);
     color: var(--nx-swatch-blue-40);
   }
 
-  &:hover {
+  &:hover:not(.active) {
     @include nx-text-helpers.semi-bold;
+    color: var(--nx-color-text-dark);
+
+    &:before {
```

Comment:
Using single-colon syntax to reference pseudo-elements is deprecated, from a time before the distinction between pseudo-elements and pseudo-classes was invented.  The double-colon syntax is the more correct way.